app/package/js/angular-multimocks.js

```diff
@@ -69,11 +70,12 @@ angular
             });
         } else {
           $httpBackend
-            .when(mock.httpMethod, mock.uri, mock.requestData)
+            .when(mock.httpMethod, uriRegExp, mock.requestData)
             .respond(mock.statusCode, mock.response, mockHeaders);
         }
 
-        // Make this http request now if required otherwise just resolve
+        // Make this HTTP request now if required otherwise just resolve
+        // TODO deprecated?
```

Comment:
This is only relevant for HAL stuff I think, and I'm not sure it's needed anymore.